seahorn/lib/fuzz_byte_buf_helper.c

```diff
@@ -0,0 +1,20 @@
+#include <assert.h>
+#include <bounds.h>
+#include <byte_buf_helper.h>
+#include <proof_allocators.h>
+
+void initialize_byte_buf(struct aws_byte_buf *const buf) {
+  size_t max_size = sea_max_buffer_size();
```

Comment:
This is what I was missing. You are not using the same initialization function as SeaHorn and KLEE. But the restructuring for KLEE made it easier to implement the functions for fuzzing! This should make it into the paper. 

I think the example of the implementation of the harnesses for all three tools is a good to add to an appendix!